nrcemt/qeels/engine/peak_detection.py

```diff
@@ -152,3 +158,99 @@ def mark_peaks(
     peak_position_y = np.round(peak_position_y)
 
     return (peak_position_x, peak_position_y, image)
+
+
+def calculation_e(E_bulk, peak_position_x):
+    difference = peak_position_x-E_bulk
```

Comment:
can you added doctoring comment here, and on other function with variables names like this.

also all of local variables should be completely lowercase. You have a lot of mixed case local variables in this file